pkgs/build-support/build-fhs-userenv-bubblewrap/env.nix

```diff
@@ -93,6 +93,28 @@ let
     paths = [ etcPkg ldconfig ] ++ basePkgs ++ targetPaths;
     extraOutputsToInstall = [ "out" "lib" "bin" ] ++ extraOutputsToInstall;
     ignoreCollisions = true;
+    postBuild = ''
+      if [[ -d  $out/share/gsettings-schemas/ ]]; then
+          # Recreate the standard schemas directory if its a symlink to make it writable
+          if [[ -L $out/share/glib-2.0 ]]; then
+              ln -s $(readlink $out/share/glib-2.0) $out/share/glib-2.0.old
+              rm -rf $out/share/glib-2.0
+              mkdir -p $out/share/glib-2.0/schemas
+          fi
+
+          # symlink any schema files or overrides to the standard schema directory
+          ln -fs $out/share/glib-2.0.old/schemas/*.xml $out/share/glib-2.0/schemas
```

Comment:
If `$out/share/glib-2.0` is not a symlink, `$out/share/glib-2.0.old` won't exist and this command either fails (if `nullopt` is unset) or creates a symlink to `$out/share/glib-2.0/schemas` in the current directory (with `nullopt` set).

Reply:
fixed, thanks for the review